examples/specs/normalized/errorbar_2d_vertical_ticks_normalized.vl.json

```diff
@@ -29,7 +29,6 @@
       "mark": {
         "type": "tick",
         "orient": "horizontal",
-        "aria": false,
```

Comment:
We should only expose one accessibility element per discrete item to screen readers. Make sure that aria false stays set.